scripts/split-e2e-tests.ts

```diff
@@ -298,6 +299,19 @@ function getRequiredJob(jobNames: string[], index: number, concurrency: number =
   }
 }
 
+/**
+ * Helper function to filter unsupported regions for certain category tests
+ * @returns list of supported regions
+ */
+function getSupportedRegions(suite: string): string[] {
+  if (suite.startsWith('src/__tests__/geo')) {
+    return AWS_REGIONS_TO_RUN_TESTS.filter(function(value, index, arr){ 
+      return value !== 'eu-west-2';
+    });
```

Comment:
would be better to filter based on existence in the geo supported regions list. That way if we add more regions to our test suite we don't have to remember to update this logic
```suggestion
    return AWS_REGIONS_TO_RUN_TESTS.filter(region => geoSupportedRegions.includes(region));
    });
```

Reply:
Changed